05-angularjs/scripts/controllers.js

```diff
@@ -0,0 +1,70 @@
+
+movieList.controller('AppCtrl', function AppCtrl ($scope) {
```

Comment:
For any specific reason are you naming your callback functions? It's not a bad practice, but not necessary, unless for debugging purposes.